tests/sentry/integrations/utils/test_code_mapping.py

```diff
@@ -262,3 +262,24 @@ def test_list_file_matches_multiple(self):
             },
         ]
         assert matches == expected_matches
+
+    def test_normalized_stack_and_source_roots_static(self):
+        stacktrace_root, source_path = self.code_mapping_helper._normalized_stack_and_source_roots(
+            "./app/", "static/app/"
+        )
```

Comment:
Unit test for `_normalized_stack_and_source_roots` function

The following tests check for different cases.